w3af/plugins/crawl/find_backdoors.py

```diff
@@ -118,27 +119,30 @@ def _check_if_exists(self, web_shell_url):
         except BaseFrameworkException:
             om.out.debug('Failed to GET webshell:' + web_shell_url)
         else:
-            signature = self._match_signature(response)
-            if signature is None:
-                return
+            if not is_404(response):
```

Comment:
I was thinking that rather than checking to make sure its not a 404 it might be better to check if it is 200:
```python
if status == 200:
```

Any thoughts? I prefer the 200 approach personally/